app/assets/stylesheets/blacklight/_constraints.scss

```diff
@@ -2,7 +2,7 @@
   @extend .mb-2;
 }
 
-span.constraints-label {
+.constraints-label {
```

Comment:
Can we just remove the style entirely? No reason to make some sr-only text bold, is there?